examples/Optical_Lumped_Amplification_Channel.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
This notebook must be improved and match the style of the other tutorial examples.

Reply:
I suggest to add a first paragraph that summarizes the notebook.

"In this notebook, you will learn how to simulate the transmission over optical fibers via the Split-Step Fourier method (SSFM).

Reply:
OK